spec/yaml/swiftnav/sbp/signing.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2022 Swift Navigation Inc.
+# Contact: https://support.swiftnav.com
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.signing
+description: Messages relating to signatures
+stable: False
+public: False
+include:
+  - types.yaml
+  - gnss.yaml
+definitions:
+  - MSG_ED25519_SIGNATURE:
+      id: 0x0BD2
```

Comment:
Not sure if there's a rule of thumb when selecting these IDs but I just chose the next available.

Reply:
let's see if we can establish a "group" of message by selecting a unique first byte so that we can have related messages grouped together by ID, for example 0xEF01, 0xEF02 (no idea if these message IDs are actually available though).

Reply:
Good idea, will do

Reply:
Claimed the `0x0C` "namespace" for these messages